setup.py

```diff
@@ -29,6 +29,9 @@
         'simplejson',
         'six>=1.4.0',
     ],
+    extras_require={
+        'uwsgi_offload': ['uWSGI>=2.0.14.0.post3'],
```

Comment:
due to the way extras work, it's best to have this be a single word like `'uwsgi'` -- also `.post3` is an internal yelp-only version, probably not a good idea for an open source project to mention it